src/extensions/default/BrambleUrlCodeHints/camera/selfieWidget.html

```diff
@@ -11,5 +11,47 @@
             <img id="selfie-photo" />
         </div>
     </div>
+    <div id="filter-list">
+      <select name="filters" id="filters-select" onchange="updateFilter()">
+          <option value="none">No Filter</option>
+          <option value="_1977">1997</option>
+          <option value="aden">Aden</option>
+          <option value="brannan">Brannan</option>
+          <option value="brooklyn">Brooklyn</option>
+          <option value="clarendon">Clarendon</option>
+          <option value="earlybird">Earlybird</option>
+          <option value="gingham">Gingham</option>
+          <option value="hudson">Hudson</option>
+          <option value="hudson">Inkwell</option>
+          <option value="kelvin">Kelvin</option>
+          <option value="lark">Lark</option>
+          <option value="lofi">Lo-Fi</option>
+          <option value="maven">Maven</option>
+          <option value="mayfair">Mayfair</option>
+          <option value="moon">Moon</option>
+          <option value="nashville">Nashville</option>
+          <option value="perpetua">Perpetua</option>
+          <option value="reyes">Reyes</option>
+          <option value="rise">Rise</option>
+          <option value="slumber">Slumber</option>
+          <option value="stinson">Stinson</option>
+          <option value="toaster">Toaster</option>
+          <option value="valencia">Valencia</option>
+          <option value="walden">Walden</option>
+          <option value="willow">Willow</option>
+          <option value="xpro2">X-pro II</option>
```

Comment:
I'll be interested in @flukeout's perspective on this, but I'd suggest that we just pick the top 6 filters, and favour those that are really different from each other.  I tested this locally, and a lot of them do basically the same thing.  We probably should simplify it for our users a bit.

That decision might also make it easier to evolve the UI so you can see all the filters at once in a grid.